im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,651 @@
+        </names>
+        <names variable="collection-editor">
+          <name form="short" et-al-min="4" et-al-subsequent-min="2"/>
+        </names>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if variable="URL">
+        <group>
+          <text variable="URL"/>
+          <group prefix=" [" suffix="]">
+            <date variable="accessed">
+              <date-part name="day" form="numeric-leading-zeros" suffix="."/>
+              <date-part name="month" form="numeric-leading-zeros" suffix="."/>
+              <date-part name="year"/>
+            </date>
+          </group>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <group>
+      <text variable="publisher"/>
+    </group>
+  </macro>
+  <macro name="year-date">
+    <choose>
+      <if variable="issued">
+        <date date-parts="year" form="text" variable="issued" prefix=" ">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="pages">
+    <choose>
+      <if type="article-journal article-magazine article-newspaper broadcast thesis webpage chapter paper-conference" match="any">
+        <label strip-periods="true" prefix=", " suffix=". " variable="page" form="short"/>
+        <text variable="page"/>
+        <text variable="note" prefix=" [erstmals " suffix="]"/>
+      </if>
+      <else-if type="entry-encyclopedia" match="any">
+        <label variable="locator" form="short" suffix=". " strip-periods="true"/>
+        <text variable="page"/>
+        <text variable="note" prefix=" [erstmals " suffix="]"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="citation-locator">
+    <choose>
+      <if type="manuscript" match="none">
+        <label variable="locator" form="short" suffix=". " strip-periods="true"/>
+      </if>
+    </choose>
+    <text variable="locator"/>
+  </macro>
+  <macro name="date">
+    <choose>
+      <if match="any" variable="issued">
+        <date form="text" variable="issued"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="collection-editor">
+    <names variable="collection-editor" prefix="Hrsg. von ">
+      <name delimiter="/" delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="volume">
+    <group suffix=".">
+      <text term="volume" form="short" suffix=" "/>
+      <text variable="volume"/>
+    </group>
+  </macro>
+  <macro name="Band aus Reihe">
+    <choose>
+      <if match="any" variable="title">
+        <group>
+          <text macro="editor" prefix=" "/>
+          <group prefix=" (" suffix=") ">
+            <group>
+              <text variable="collection-title"/>
+              <choose>
+                <if match="any" variable="collection-editor">
+                  <text value=". "/>
```

Comment:
don't put punctuation in text values, that's ugly and fragile. Instead, use a group delimiter:

```
<group delimiter=". ">
  <text variable="collection-title"/>
  <text macro="collection-editor"/>
</group>
```